EndToEndTests/CustomersAndOrdersEndToEndTest.cs

```diff
@@ -8,13 +8,14 @@
 using ServiceCommon.Common;
 using ServiceCommon.Classes;
 using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion.Internal;
+using System.Threading;
 
 namespace EndToEndTests
 {
     [TestClass]
     public class CustomersAndOrdersEndToEndTest
     {
-        long uniqueId = System.DateTime.Now.Ticks;
         string hostCustmer = "http://localhost:8081/";
```

Comment:
customer is mispelt.
Neither of these values are hosts.
They are baseUrls

Also, is `order` always appended to hostOrder and customer always appended to hostCustomer. If so why not do that here.